src/Command/RecipesCommand.php

```diff
@@ -140,16 +142,31 @@ private function displayPackageInformation(Recipe $recipe)
             $status = '<comment>update available</comment>';
         }
 
+        $gitSha = $recipeLock['recipe']['branch'] ?? 'master';
+        if (null !== $lockRef) {
+            $gitSha = $this->findRecipeShaFromTreeRef(
+                $recipe->getName(),
+                $recipeLock['recipe']['repo'],
+                $recipeLock['recipe']['branch'],
+                $recipeLock['version'],
+                $lockRef
+            );
+        }
+
         $io = $this->getIO();
-        $io->write('<info>name</info>     : '.$recipe->getName());
-        $io->write('<info>version</info>  : '.$recipeLock['version']);
+        $io->write('<info>name</info>             : '.$recipe->getName());
+        $io->write('<info>version</info>          : '.$recipeLock['version']);
+        $io->write('<info>status</info>           : '.$status);
         if (!$recipe->isAuto()) {
```

Comment:
We can add if $gitSha is not null

Reply:
If we can't figure out the `$gitSha`, I default it to the branch from the lock file (e.g. `master`) so that we can at least give an approximation. So I'd like to keep this as always showing (except when the recipe is an auto recipe)